### ISSUE TITLE
Errors that cause load to fail with single core usage doesn't result in failure under multicore mode

### ISSUE THREAD

**Issue by dgaston** [arq5x/gemini#406]
I posted in the Google Group as well concerning why the IndexError happens in the first place (see here: https://groups.google.com/forum/#!topic/gemini-variation/gvXLcPQVlRc) as that may be a problem with my VCF. However the fact that when in running in multicore mode the load process completes is another issue. While the errors should be apparent in the output a user might just assume they aren't impacting the final results. If they don't look too closely at their output they might not notice they are missing the majority of their data in the database.

**Comment by brentp** [comment]
yes, I've seen this as well. If any of the cores have an error, we should fail sooner and remove the tmp files.
